qa/rpc-tests/sc_quality_nodes.py

```diff
@@ -37,21 +37,6 @@ def setup_network(self, split=False):
         sync_blocks(self.nodes[1:NUMB_OF_NODES])
         sync_mempools(self.nodes[1:NUMB_OF_NODES])
         self.is_network_split = split
```

Comment:
I haven't re-run this test, but I think that removing the `sync_all()` at the end of the `setup_network()` is not safe

Reply:
I removed this function, cause it can be inherited.

Reply:
Sorry, maybe my comment was a bit confusing, as I was referring to the last instruction of the `setup_network()` defined here: there was a `self.sync_all()` that has been removed, I don't know if it is intentional

Reply:
The setup_network is derived from the base class. Now, it's only setup_node overidden.

Reply:
Oh, I see! I was confused by the renamed function, thanks!